### PR TITLE
React capstone project: Metric webapp

### PR DESCRIPTION
**Pull request by Bhekisile**
## This repository contains

- Setup start-up files
- Created a navigation router with Links
- Attached Stock Screener website [video](https://www.loom.com/share/3623b2760a334f94a12b42b8f8e970bf)
- Fetched data from external API and stored it in the redux store
- Display the list of companies with each company's details 
- When the user clicks on the company's card a page with screeners' name and their corresponding values displays
- Implemented a back button which allows the user to navigate to the home page
- Created search button to filter the data by company name 
- Added tests for pure functions
- Updated README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,140 +26,79 @@ For more information on the importance of a professional README for your reposit
...
+- **[Display screeners details]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+Coming soon
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] kindly deploy your app and add a link to the deployed app

File: src/components/Company.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    {screener.companyName}
+                  </li>
+                  <li>{screener.sector}</li>
+                </ul>
+              </Link>
+            </div>
+          ))}
+      </div>
+      <Footer />
+    </div>
```
Comment (VICTOR-CHUKWU) [inline src/components/Company.js:61]: - [ ] Kindly note that according to project requirements, you are to keep to typography, spacing and generally the layouts, currently Your cards are stacked in a one-column grid in the mobile view, but the design uses a two-column grid, also There is no space between cards in the given design while yours have spaces between, kindly fix this
your design :point_down: 
Screenshot from 2023-09-06 11-03-30
given design :point_down: 
Screenshot from 2023-04-28 11-04-25

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (VICTOR-CHUKWU) [inline src/App.css]: - [ ] you already have this style in your index.css, kindly remove here to make your code base cleaner

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Bhekisile , 

## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/styles/Company.css [hunk 4]
```diff
@@ -31,16 +30,22 @@ body {
...
 }
 
+/* .company-card:nth-child(8n + 2),
+.company-card:nth-child(8n + 4),
+.company-card:nth-child(8n + 5),
+.company-card:nth-child(8n + 7) {
+  background-color: rgb(197, 217, 237);
+  border: 2px solid #fff;
+} */
+
```
Comment (zieeco) [inline src/components/styles/Company.css]: - Hey, great job addressing all the issues pointed out by the previous reviewer, however, you've added these commented pieces of code that shouldn't be in your code base as they make your codebase not clean and maintainable. Please remove them, as this is the only reason your project won't be approved.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Bhekisile,




Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice design. ✔️
- Descriptive PR. ✔️
- Great job implementing the requested changes ✔️



## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/Company.css (hunk 4 above)
Comment (Bhekisile) [reply src/components/styles/Company.css]: I've made changes. Thank you

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Bhekisile, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**